README.md

```diff
@@ -868,6 +868,112 @@ scala> val sum = List(USD(100), USD(10)).sum
+SI, such as litre, hectare, hour, minute, etc). See the linked document for a detailed list.
+
+To summon the strict SI `UnitGroup` for `Length`, you would use this code:
+
+```scala
+import squants.space.Length
+// import squants.space.Length
+
+import squants.unitgroups.ImplicitDimensions.space._
+// import squants.unitgroups.ImplicitDimensions.space._
+
+import squants.unitgroups.UnitGroup
+// import squants.unitgroups.UnitGroup
+
+import squants.unitgroups.si.strict.implicits._
+// import squants.unitgroups.si.strict.implicits._
+
+val siLengths: UnitGroup[Length] = implicitly[UnitGroup[Length]]
+// siLengths: squants.unitgroups.UnitGroup[squants.space.Length] = squants.unitgroups.si.strict.package$implicits$$anon$1@1045e253
+```
+
+To print out units and their conversion factors to the primary SI unit, you could use this code:
+
+```scala
+import squants.{Quantity, UnitOfMeasure}
+// import squants.{Quantity, UnitOfMeasure}
+
+def mkConversionFactor[A <: Quantity[A]](uom: UnitOfMeasure[A]): Double = {
+  val one = uom(1)
+  one.to(one.dimension.siUnit)
+}
+// mkConversionFactor: [A <: squants.Quantity[A]](uom: squants.UnitOfMeasure[A])Double
+
+def mkTuple[A <: Quantity[A]](uom: UnitOfMeasure[A]): (String, Double) = {
+  (uom.symbol, mkConversionFactor(uom))
+}
+// mkTuple: [A <: squants.Quantity[A]](uom: squants.UnitOfMeasure[A])(String, Double)
+
+siLengths.sortedUnits.toList.map(mkTuple).foreach(println)
+// (nm,1.0E-9)
+// (µm,1.0E-6)
+// (mm,0.001)
+// (cm,0.01)
+// (dm,0.1)
+// (m,1.0)
+// (dam,10.0)
+// (hm,100.0)
+// (km,1000.0)
+```
+
+Note that `UnitGroup`'s `sortedUnits` field is a `SortedSet`, so before mapping over it,
+you will probably want to convert it to a List, otherwise the output may be resorted.
+
+### Non-SI UnitGroups
+
+Other `UnitGroup` definitions don't use implicits. For example, `squants.unitgroups.uscustomary.space.UsCustomaryLiquidVolumes` or `squants.unitgroups.misc.TroyMasses` can be imported and used directly.
+
+### Creating an ad-hoc UnitGroup
+
+To create an ad-hoc `UnitGroup` just implement the trait. For example, to make a US cooking measure `UnitGroup`:
+
+```scala
+import squants.{Quantity, Dimension}
+// import squants.{Quantity, Dimension}
+
+import squants.space._
+// import squants.space._
+
+import squants.unitgroups.UnitGroup
+// import squants.unitgroups.UnitGroup
+
+val usCookingUnitGroup = new UnitGroup[Volume] { 
+  // units don't have to be specified in-order.
+  val units: Set[UnitOfMeasure[Volume]] = Set(UsPints, UsGallons, Teaspoons, Tablespoons, UsQuarts, FluidOunces)
+}
+// usCookingUnitGroup: squants.unitgroups.UnitGroup[squants.space.Volume]{val units: Set[squants.UnitOfMeasure[squants.space.Volume]]} = $anon$1@25f41c10
+
+// squants automatically sorts units
+usCookingUnitGroup.sortedUnits.foreach(println)
+// squants.space.Teaspoons$@797a286f
```

Comment:
Because this output includes memory locations, this part of the README will change every time tut is run.

As a future enhancement we should provide a stable `toString` for `UnitOfMeasure` that outputs the unit's name.